tests/test_modeling_tf_common.py

```diff
@@ -276,12 +276,19 @@ def test_onnx_runtime_optimize(self):
     def test_keras_save_load(self):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
 
+        # `CLIPModelTester/TFCLIPModelTester` set `return_loss` to `True`, however this causes the preparation
+        # of `symbolic_inputs` below failed.
+        if self.__class__.__name__ == "TFCLIPModelTest":
+            inputs_dict.pop("return_loss", None)
+
         tf_main_layer_classes = set(
             module_member
             for model_class in self.all_model_classes
             for module in (import_module(model_class.__module__),)
             for module_member_name in dir(module)
             if module_member_name.endswith("MainLayer")
+            # This condition is required, since `modeling_tf_clip.py` has 3 classes whose names end with `MainLayer`.
+            and module_member_name[: -len("MainLayer")] == model_class.__name__[: -len("Model")]
```

Comment:
Can we group the two together with a single
```
if module_member_name.replace("MainLayer", "Model") == model_class.__name__
```

Reply:
if `module_member_name` itself is something like `TFCLIP(Text/Vision)Model`, the condition in your comment will hold, but we only want to deal with `TFCLIP(Text/Vision)MainLayer`, right?